cockroachcloud/cluster-management.md

```diff
@@ -52,6 +52,10 @@ From the **Overview** page, you can connect to your cluster. For more informatio
 
 ## Scale your cluster
 
+{{site.data.alerts.callout_info}}
+During [limited access](/docs/{{site.versions["stable"]}}/cockroachdb-feature-availability.html), {{ site.data.products.dedicated }} clusters on Azure cannot be scaled. Refer to [{{ site.data.products.dedicated }} on Azure (cockroachdb-dedicated-on-azure.html).
```

Comment:
Will have to wait for @adwittumuluri to confirm on the customer support route, but I think for public doc for limited access, we could let it be since it's expected that customers recreate clusters when needed to scale and that should be ok since it's not production.